pheno_utils/pheno_loader.py

```diff
@@ -468,13 +468,16 @@ def get(self, fields: Union[str,List[str]], flexible: bool=None, not_bulk_field=
             
             if table_name == 'age_sex':
                 keep_undefined = True
```

Comment:
why do we keep undefined here? can't this lead to a cartesian product if there are multiple undefined rows for a participant? (maybe there's a good reason, just noticed this and started thinking)

Reply:
The reason for this is that in the 'age_sex' df, we don't have an undefined research_stage. So, I want to keep the undefined research stage on the left side of the join because we know that in this specific case, it couldn't lead to a Cartesian product.

Reply:
Great, thanks! Worth a short comment in the code.